src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -21,12 +22,14 @@
   private static Jedis node2;
   private static Jedis node3;
   private static Jedis node4;
+  private static Jedis nodeSlave2;
   private String localHost = "127.0.0.1";
 
   private HostAndPort nodeInfo1 = HostAndPortUtil.getClusterServers().get(0);
   private HostAndPort nodeInfo2 = HostAndPortUtil.getClusterServers().get(1);
   private HostAndPort nodeInfo3 = HostAndPortUtil.getClusterServers().get(2);
   private HostAndPort nodeInfo4 = HostAndPortUtil.getClusterServers().get(3);
+  private HostAndPort nodeInfoSlave2 = HostAndPortUtil.getClusterServers().get(4);
```

Comment:
For what I see this is not a `slave` node of the cluster. It's another master node...